src/s3b_the_roach_market.tw2

```diff
@@ -0,0 +1,278 @@
+
+::RM-DopemanDan-blocks-path
+“Whoa whoa whoa, what’s the damage, $playerName? What I done to deserve this, other than be the number-one hookup for anything and everything this side of Gatherville?” Dan said. “You cuss out Santa like that too, when he come down your chimney?”
+
+"Who the hell you ever seen got a tarp with a chimney, Dan?" I sassed back. Then I sighed. “Dan, whatever you’re slinging— just leave me the fuck alone, dude. Jaysus.”
+
+Dan rolled his eyes. “Look, ‘lette, I ain’t lyin’. Dopeman’s got some shit with your name on it, see?” He fished a brown paper bag out of his pocket and dangled it in front of me. <i><b>PICKUP: $playerName</b></i> was written on it in black marker. “And I never known nobody to turn down shit that was already paid for.”
+
+“Uhh, that isn’t mine, dude. Must be some other $playerName came by here. [[Go bug them, okay?|RM-DopemanDan-choice]]”
+
+::RM-DopemanDan-choice
+“Nah nah nah, it’s you, girl. Just how many $playerNames you think I know? Anyway, I know you didn’t buy this shit; Anna did. Said she’d send you by to pick it up, and here you are, so—” he thrust the bag toward my arms— “pick this shit <i>up</i>, girl. Leave Dopeman Dan in peace, all right? Goddamn!”
+
+> [[“Fine, great. I’ll play drug mule for you and Anna, just leave me alone.”|RM-DopemanDan-leave-alone]]
+> [[“That doesn’t sound like Anna, but whatever. Hand ‘em over. Free drugs is free drugs.”|RM-DopemanDan-free-drugs]]
+
+<!-- TODO: either adds “drugs” to inventory -->
+
+::RM-DopemanDan-leave-alone
+I snatched the bag out of Dan’s hand, and stepped aside to push by him yet again. This time he didn’t try to stop me.
+
+“See? Was that so hard? Maybe next time you just take Dopeman at his word and save the attitude for somebody deserves it.”
+
+I was already past him, and waved my hand dismissively without looking back. “Yeah sure Dan, sorry or whatever. I’ll see ya around.”
+
+> [[Head to Auntie’s gift shop|RM-giftshop-entrance]]
+> [[Head to the junk heap|RM-junkheap-entrance]]
+
+::RM-DopemanDan-free-drugs
+Dan cackled. “Hell yeah! Now you’re talking words Dopeman understands.” He thrust the bag forward, and I took it.
+
+I unrolled the top and peeked inside. “Wow. This is, uh, a pretty full bag.”
+
+“Yeah, dawg. I figure she was either getting herself set up for a sweet-ass month or one fuckin’ buckwild night.”
+
+I was about to ask Dan if this was a regular thing for Anna, but then I thought better of it. I didn’t want him to think he had anything to say about her that I didn’t already know.
+
+“Well, cool, man. With a haul like this, I bet she’d never notice if a little went missing.” I winked at him.
+
+Dan tapped the side of his nose with his finger. “Trip in peace, my ‘lette. Dopeman ain’t never snitched on nobody.”
+
+I nodded, and made my way deeper into the Roach Market.
```

Comment:
Maybe it's the default text style, but this looks really dense when it appears all at once. How do you feel about using `click:` to break this up a bit?

Reply:
Yeah, I def plan on splitting this one up into two chunks.